strapi/documentation
# Fix table overflow behavior on mobile

### What does it do?

Updated scss for `table` element so that table overflow doesn't push screen out of bounds on mobile.

### Why is it needed?

UI bug that should be fixed for UX. This happens across the entire documentation sites, whenever we have `<code>` wrapped text in the tables that are too long, and pushes the table out of the viewport bounds.

### Changes Made
- Changed table overflow to `auto`
- Removed comments since `<Annotations>` usage isn't affected by initial or auto (see #How to test)

### Screenshots & gifs

**Before Fix - iPhone 14 Pro Max Example**
This was tested on an iPhone 14 plus in real life as well.
Link: https://docs.strapi.io/dev-docs/backend-customization/requests-responses

<table>
  <tr>
    <th>Before - UI bug</th>
    <th>After - overflow, scrollable</th>
  </tr>
  <tr>
    <td><img src="https://github.com/strapi/documentation/assets/111836326/b15c9ec9-b9a5-4147-96a6-9cfd8d103dcb" width="300px"/></td>
    <td><img src="https://github.com/strapi/documentation/assets/111836326/97ac5fc6-2c83-40e6-aec8-a69780193219" width="280px"/></td>
  </tr>
</table>

### How to test
1. Visit a page with many long tables like https://docs.strapi.io/dev-docs/backend-customization/requests-responses
2. Test adding `<Annotation>` to a markdown table to make sure that changes don't affect it's usage

<img src="https://github.com/strapi/documentation/assets/111836326/48f83d2c-7d44-412f-8a11-26cc9ce10a12" width="200px" />

### Related issue(s)/PR(s)

\-
